### ISSUE TITLE
latest version in npm is broken bc of zuul-ngrok private repo devDep

### ISSUE THREAD

**Issue by cmawhorter** [auth0/lock#882]
it looks like #874 fixed this in the repo but the latest version published on npm still has a devDep pointing to a private repo, which causes builds to fail.

any chance npm could be bumped to fix this?

also -- any chance of publishing the [compiled dist lock bundles](https://github.com/rollup/rollup/wiki/pkg.module)?

**Comment by hzalaz** [comment]
Will be fixed in the next release.

Also we are not planning on distributing lock list bundle in npm since its already available in the cdn but we might revisit this later

**Comment by hzalaz** [comment]
BTW, release will be tomorrow EOD

**Comment by cmawhorter** [comment]
the reason i mentioned shipping dist with npm is because i've found it to be a non-trivial issue work with building npm deps.  

npm v4 (3 too?) doesn't install devDeps for deps, so you need to add a convoluted postinstall script that `cd`s into node_modules/auth0-lock and `npm install` and blah blah blah.  And then some deps (not sure if lock is among them) won't build because "node_modules" is on the path, and most webpack.config files exclude via a broad match `/node_modules/` regex.

tl;dr;  i've found it be to surprisingly difficult to manage unbuilt deps with npm.
